dispatchtable.js

```diff
@@ -62,13 +62,13 @@ var dispatch_table = {
     },
     "!sheetattack": {
         "action": (args, msg) => { handle_sheetattack(args, msg); },
-        "re_syntax": /^(!sheetattack|!attack) [-_a-zA-Z0-9]{20} (mid|low|high) (H|B|E|P|F) (melee|missile) [-+]?[0-9]+ [-_a-zA-Z0-9]{20} .+$/,
-        "hr_syntax": "![sheet]attack attacker_id (high|low|mid) (H|B|E|P) (missile|melee) modifier defender_id weapon"
+        "re_syntax": /^(!sheetattack|!attack) [-_a-zA-Z0-9]{20} (high|mid|low|arms|legs|torso|head|neck|skull|abdomen|face|thorax|shoulder|hip|thigh|knee|calf|foot|upper_arm|elbow|forearm|hand|groin) (H|B|E|P|F) (melee|missile) [-+]?[0-9]+ [-_a-zA-Z0-9]{20} .+$/,
+        "hr_syntax": "![sheet]attack attacker_id (high|mid|low|arms|legs|torso|head|neck|skull|abdomen|face|thorax|shoulder|hip|thigh|knee|calf|foot|upper_arm|elbow|forearm|hand|groin) (H|B|E|P) (missile|melee) modifier defender_id weapon"
     },
     "!attack": {
         "action": (args, msg) => { handle_attack(args, msg); },
-        "re_syntax": /^(!sheetattack|!attack) [-_a-zA-Z0-9]{20} (mid|low|high) (H|B|E|P|F) (melee|missile) [-+]?[0-9]+ [-_a-zA-Z0-9]{20} .+$/,
-        "hr_syntax": "![sheet]attack attacker_id (high|low|mid) (H|B|E|P) (missile|melee) modifier defender_id weapon"
+        "re_syntax": /^(!sheetattack|!attack) [-_a-zA-Z0-9]{20} (high|mid|low|arms|legs|torso|head|neck|skull|abdomen|face|thorax|shoulder|hip|thigh|knee|calf|foot|upper_arm|elbow|forearm|hand|groin) (H|B|E|P|F) (melee|missile) [-+]?[0-9]+ [-_a-zA-Z0-9]{20} .+$/,
```

Comment:
Thanks for adding this. It will be a good thing to add behind a house rule flag.